cmd/soroban-rpc/internal/methods/get_ledger_entries.go

```diff
@@ -0,0 +1,108 @@
+package methods
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/code"
+	"github.com/creachadair/jrpc2/handler"
+
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/go/xdr"
+
+	"github.com/stellar/soroban-tools/cmd/soroban-rpc/internal/db"
+)
+
+type GetLedgerEntriesRequest struct {
+	Keys []string `json:"keys"`
+}
+
+type LedgerEntryResult struct {
+	Found              bool   `json:"found"`
```

Comment:
Wasn't sure how to cleanly have "null" as a result (not sure if changing `Entries` in `GetLedgerEntriesResponse` to type `[]*LedgerEntryResult` is the way to go), so I went with adding this `Found` field which clients should check is True before processing an entry. Open to suggestions on how to handle this.

Reply:
could keep it this way, or another way to consider from a restful resource pattern when query is by list of ids, it's more a search then a getter and response can just contain entries that matched, ranging from an empty list to whatever. it won't indicate 'not found' per id, but client can derive that on their own if it needed. I think if you want that signal in the response model, another way to accomplish it over json-rpc would be:
```
GetLedgerEntryRequest struct {
   Key string `json:"key"`
}
```
And client submits multiple of these in a json-rpc batch request, that way client gets a response for each requested id in a batch result.

Reply:
Both good points. I went with replacing the `Found` field in `LedgerEntryResult` with a `Key` field, and only including present ledger entries in the final `GetLedgerEntriesResponse`